ocw/lib/azure.py

```diff
@@ -137,7 +135,7 @@ def container_valid_for_cleanup(container) -> bool:
         '''
         if Instance.TAG_IGNORE in container['metadata']:
             return False
-        if re.match('^bootdiagnostics-', container.name):
+        if container.name.startswith("bootdiagnostics-"):
```

Comment:
for me it is the same , why `startswith` is better ?

Reply:
> for me it is the same , why `startswith` is better ?

It's more Pythonic using a string method instead of the `re` module which makes it Perlish. ;)